Composer/packages/client/src/components/Page.tsx

```diff
@@ -5,11 +5,10 @@
 import { jsx, css, SerializedStyles } from '@emotion/core';
 import React from 'react';
 import { FontWeights, FontSizes } from 'office-ui-fabric-react/lib/Styling';
-import { useRecoilValue } from 'recoil';
 
 import { LeftRightSplit } from '../components/Split/LeftRightSplit';
 import { navigateTo, buildURL } from '../utils/navigation';
-import { currentModeState, rootBotProjectIdSelector } from '../recoilModel';
+import { PageMode } from '../recoilModel';
```

Comment:
Should this just be moved into this file?

Reply:
I'm leaving PageMode where it is because it still gets used in the Recoil model as a list of keys into the pageElementState atom.